pytest/index_signature_request.py

```diff
@@ -1,298 +0,0 @@
-
-    # Set up the node's home directories
-    for i in mpc_nodes:
-        # Move the generated mpc configs
-        mpc_config_dir = dot_near / str(i - num_validators)
-        for fname in os.listdir(mpc_config_dir):
-            subprocess.run(('mv', os.path.join(mpc_config_dir,
-                                               fname), nodes[i].node_dir))
-
-        # Indexer config must explicitly specify tracked shard
-        fname = os.path.join(nodes[i].node_dir, 'config.json')
-        with open(fname) as fd:
-            config_json = json.load(fd)
-        config_json['tracked_shards'] = [0]
-        with open(fname, 'w') as fd:
-            json.dump(config_json, fd, indent=2)
-        print(f"Wrote {fname} as config for node {i}")
-
-        # Create respond.yaml with credentials for sending responses
-        account_id = f"respond.test{i}"
-        access_keys = [Key.from_seed_testonly(account_id, seed=f"{s}")
-                       for s in range(0, num_respond_aks)]
-        tx = sign_create_account_with_multiple_access_keys_tx(
-                nodes[i].signer_key, account_id, access_keys, 1, last_block_hash)
-        send_and_confirm_tx(tx)
-        respond_cfg = {
-            'account_id': account_id,
-            'access_keys': list(map(serialize_key, access_keys)),
-        }
-        fname = os.path.join(nodes[i].node_dir, 'respond.yaml')
-        with open(fname, "w") as file:
-            yaml.dump(respond_cfg, file, default_flow_style=False)
-
-    def secret_key_hex(i):
-        return str(chr(ord('A') + i) * 32)
-
-    def p2p_private_key(i):
-        return open(pathlib.Path(nodes[i].node_dir) / 'p2p_key').read()
-
-    # Deploy the mpc contract
-    tx = sign_deploy_contract_tx(nodes[0].signer_key, load_mpc_contract(), 10,
-                                 last_block_hash)
-    send_and_confirm_tx(tx)
-
-    # Initialize the mpc contract
-    init_args = {
-        'threshold': num_mpc_nodes,
-        'candidates': participants,
-    }
-    tx = sign_function_call_tx(nodes[0].signer_key,
-                               nodes[0].signer_key.account_id, 'init',
-                               json.dumps(init_args).encode('utf-8'),
-                               150 * TGAS, 0, 20, last_block_hash)
-    res = nodes[0].send_tx_and_wait(tx, 20)
-    assert ('SuccessValue' in res['result']['status'])
-
-    # Start the mpc nodes
-    for i in mpc_nodes:
-        cmd = (mpc_binary_path, 'start', '--home-dir', nodes[i].node_dir)
-        # mpc-node produces way too much output if we run with debug logs
-        nodes[i].run_cmd(cmd=cmd,
-                         extra_env={
-                             'RUST_LOG': 'INFO',
-                             'MPC_SECRET_STORE_KEY': secret_key_hex(i),
-                             'MPC_P2P_PRIVATE_KEY': p2p_private_key(i),
-                             'MPC_ACCOUNT_SK': near_secret_key(i),
-                         })
-
-    return nodes
```

Comment:
this was moved to `common_lib/shared.py` and renamend: 
https://github.com/Near-One/mpc/blob/5bd9f412c571e44751e8c11c387ccb752cd1464a/pytest/common_lib/shared.py#L248